odl/discr/discr_mappings.py

```diff
@@ -138,15 +138,28 @@ def order(self):
         return self._order
 
 
-class RawGridCollocation(FunctionSetMapping):
+class GridCollocation(FunctionSetMapping):
 
     """Function evaluation at grid points.
 
-    This is the raw `Operator` version of the default 'restriction'
-    used by all core discretization classes.
+    Given points :math:`x_1, \dots, x_n \\in \Omega \subset \mathbb{R}^d`,
```

Comment:
In general, I think we should keep the math parts like this in the doc to a minimum. It is quite hard to read. Instead try to fit code in the docstrings, and have the math in the online documentation?